components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -920,226 +919,46 @@ SUBROUTINE p3_main(qc,nc,qr,nr,th,qv,dt,qitot,qirim,nitot,birim,   &
 
           !................................................................
           ! deposition/condensation-freezing nucleation
-          ! allow ice nucleation if < -15 C and > 5% ice supersaturation
```

Comment:
If we agree that calc_xx_aaa (line 904 above) is also no longer needed, per the comment we could remove it as part of this PR.  Just to have some cleaner code.

Reply:
Sure, I'll remove this.

Reply:
We should also remove the subroutine itself, starting at line 3662 I believe.